.github/workflows/build-all.yml

```
@@ -0,0 +1,160 @@
+      - '.github/workflows/**'
+
+jobs:
+  build-linux:
+    name: Build Linux ${{ matrix.arch }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)